README.md

```diff
@@ -1,22 +1,20 @@
 # Llop
-Polls made easy
-Llop is a multi-feature web application that assists users in decision making using a simple voting poll interface. This project incorporates JavaScript, HTML, CSS, jQuery, AJAX front-end skills, and Node, Express, PostgresSQL, and Knex back-end skills. 
+"Llop, polls made easy" is a multi-feature web application that assists users in decision making using a simple voting poll interface. This project incorporates JavaScript, HTML, CSS, jQuery, and AJAX as the front-end skills. And it combines Node, Express, PostgresSQL, and Knex as the  back-end skills. 
 
 ## Features
-1. Users can create a poll by entering a valid email address as well as their name.
+1. Users can create a poll by entering a valid email address and their name.
 2. Poll creators must enter a question and at least two options in order to successfully create a poll.
-3. Upon poll creation, two links are generated: to the results page and the voting page. These links are then sent to the poll creator via email.
-4. Poll creators can then share the voting link with their friends to invite them to the poll.
+3. Two links will be generated and sent to the inbox of the poll creaters after a poll creation: one is the results page, and the other one is the voting page.
+4. Poll creators can share the voting link with their friends to invite them to the poll.
 5. On the poll page, voters can rank their choices using the drag and drop feature.
-6. When a vote has been submitted, the poll creator will receive an email with the vote detail and a link to the results page.
+6. When a vote has been submitted, the poll creator will receive an email that gives the voting detail and a link to the results page.
 
 ## Getting Started 
-1. Fork this repository, then clone your fork of this repository.
-2. Install dependencies using the `npm install` command.
-3. Create a `.env` file and put in configurations according to `.example.env` (you need a valid sendgrid API key to get the email feature working).
-3. Start the web server using the `npm run local` command. The app will be served as you configurated.
-4. Go to the address in your browser.
-5. Alternatively go to <http://llop.herokuapp.com/>.
+1. Fork this repository and create a local clone of your fork.
+2. Install the dependencies(described at the end) using the `npm install` command.
+3. Follow the format of `.example.env`, then create a `.env` file and put it into the configurations(you need a valid sendgrid API key to get the email feature working).
+3. Start the web server using the `npm run local` command. The app will be served as it is configurated.
+4. Go to the address of your browser or go to <http://llop.herokuapp.com/>.
```

Comment:
Go to the address `localhost:8080` in your browser or go to <http://llop.herokuapp.com/>.

Reply:
Hi, I changed this already. Thank you.